app/views/docs/email-and-sms-templates.phtml

```diff
@@ -0,0 +1,280 @@
+<p>Appwrite uses email and SMS messages to communicate with users to perform authentication actions. Emails and SMS messages can be customized to fit your app's design and voice.</p>
+
+<p>
+    Each Appwrite project can have its own set of unique templates. 
+    Templates also support localization, every template can be written in multiple languages and served 
+    depending on the configured locale.
+</p>
+
+<p>
+<a href="#email-templates"><i class="icon-angle-circled-right"></i>Email Templates</a>
+</p>
+<p>
+<a href="#sms-templates"><i class="icon-angle-circled-right"></i>SMS Templates</a>
+</p>
+
+<h2><a href="#customize" id="customize">Customize Templates</a></h2>
+<p>You can customize email templates for each of your projects in the Appwrite console. </p>
+<div class="notice">
+    <h2>Custom SMTP Server Required</h2>
+    <p>The built-in email service does not support custom email templates. Configure a <a href="#smtp">custom SMTP server</a> to enable custom email templates.</p>
```

Comment:
```suggestion
    <p>The built-in email service does not support custom email templates to prevent malicious templates. Configure a <a href="#smtp">custom SMTP server</a> to enable custom email templates.</p>
```